### PR TITLE
Create Power BI Pro Licenses.pbix

### PR DESCRIPTION
**Pull request by mmaclean13**
test action by committing .pbix file

### CODE REVIEW

**Review by nathangiusti — CHANGES_REQUESTED** [review]
The action did not run correctly. The workflow does not contain a trigger event. 

Take a look at this: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions
Let me know if you have questions.

**Comment by mmaclean13** [comment]
@nathangiusti - just updated the action to contain an **on: [pull request]** trigger. Would you please take a look?

File: .github/workflows/format_json.yaml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 name: 'PBIX Deserializer'
 description: 'Deserializes PBIX Files into json files for easier version control'
+on: [pull request]
```
Comment (nathangiusti) [inline .github/workflows/format_json.yaml:3]: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpull_requestpull_request_targetbranchesbranches-ignore

**Review by nathangiusti — CHANGES_REQUESTED** [review]
Still have an error. Look at the actions tab of your repo and check out the error message from the latest run and see if you can figure out the fix. If you can't ping me and we can work it out together.

File: .github/workflows/format_json.yaml [hunk 2]
```diff
@@ -1,7 +1,9 @@
 name: 'PBIX Deserializer'
 description: 'Deserializes PBIX Files into json files for easier version control'
-runs:
-  using: "composite"
+on: pull_request
+jobs:
+    Reformat-PBIX-Files:
+runs-on: ubuntu-latest
```
Comment (nathangiusti) [inline .github/workflows/format_json.yaml:6]: The tabbing is off, that's why it's crashing. Yaml is very white space sensitive.

You can paste your yaml here to verify that it is formatted correctly. http://www.yamllint.com/

**Review by nathangiusti — CHANGES_REQUESTED** [review]

File: .github/workflows/format_json.yaml (hunk 2 above)
Comment (nathangiusti) [reply .github/workflows/format_json.yaml:6]: Python is also very white space sensitive so you'll need to get used to it.

**Review by nathangiusti — CHANGES_REQUESTED** [review]
The error has to do with the description. Check and see if it matches the way it's laid out in other workflows. I won't be able to take a look at this until tomorrow.

**Comment by mmaclean13** [comment]
It ran!

File: .github/workflows/format_json.yaml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 name: 'PBIX Deserializer'
-description: 'Deserializes PBIX Files into json files for easier version control'
+#description: 'Deserializes PBIX Files into json files for easier version control'
```
Comment (nathangiusti) [inline .github/workflows/format_json.yaml:2]: Workflows don't have a description field. Go ahead and delete this line and we can look at deploying it to all of our repos.

**Review by nathangiusti — CHANGES_REQUESTED** [review]

**Review by nathangiusti — APPROVED** [review]

**Pull request merged**